settings.gradle

```diff
@@ -1 +1,6 @@
 rootProject.name = 'tic-tac-toe'
+
+include 'battle-service'
+include 'statistics-service'
+include 'user-service'
+include 'common'
```

Comment:
let's probably remove common module. we don't know so far if we will need this at all.
also "common" is a very wide term ,which actually doesn't help to solve specific problem, but rise fault wished to put as much as possible there. which usually leads to more problems.